tests/src/test/scala/cats/tests/KleisliSuite.scala

```diff
@@ -39,13 +39,13 @@ class KleisliSuite extends CatsSuite {
   checkAll("Semigroupal[Kleisli[Option, Int, ?]]", SerializableTests.serializable(Semigroupal[Kleisli[Option, Int, ?]]))
 
```

Comment:
can we revert the style changes? I'd prefer a PR with style only changes so we can focus on logic/design.